client/layout/masterbar/style.scss

```diff
@@ -75,15 +75,15 @@ body.is-mobile-app-view {
 
 			.masterbar__item-content {
 				font-weight: 600;
-				color: var(--color-sidebar-text);
+				color: var(--studio-gray-100);
```

Comment:
Just checking that these don't apply somewhere we don't expect them to. This seems not just for the reader but more generally across the logged out masterbar?

Il test a little around the other logged-out pages to see if there are any unexpected changes. But if this is propagating to more places than we want, we might just redefine `--color-sidebar-text` as `--studio-gray-100` within the context of the reader pages only.

Reply:
We seem good. I cant find much of anything and it seems most of the logged out outside of this don't use the .masterbar__loggedout class.

Reply:
Great, thanks for taking a look! Will go ahead and merge this 🙏